concent_api/core/views.py

```diff
@@ -554,6 +425,23 @@ def handle_send_force_get_task_result(request, client_message: message.concents.
             requestor_public_key = client_public_key,
             status               = ReceiveStatus,
         )
+        subtask = store_or_update_subtask(
+            task_id                     = client_message.report_computed_task.task_to_compute.compute_task_def['task_id'],
+            subtask_id                  = client_message.report_computed_task.task_to_compute.compute_task_def['subtask_id'],
+            provider_public_key         = other_party_public_key,
+            requestor_public_key        = client_public_key,
+            state                       = Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
+            next_deadline               = client_message.report_computed_task.timestamp + settings.CONCENT_MESSAGING_TIME,  # TODO: Is it ok ??
+            force_get_task_result       = client_message,
+            report_computed_task        = client_message.report_computed_task,
+            task_to_compute             = client_message.report_computed_task.task_to_compute,
+        )
+        store_pending_message(
+            response_type       = PendingResponse.ResponseType.ForceGetTaskResult,
+            client_public_key   = other_party_public_key,
```

Comment:
Provider should get `ForceGetTaskResultUpload` (which contains `ForceGetTaskResult`), not `ForceGetTaskResult` alone.

Reply:
And he got. Concent never returns `ForceGetTaskResult` alone
```python
        response_to_client = message.concents.ForceGetTaskResultUpload(
            file_transfer_token     = file_transfer_token,
            force_get_task_result   = message.concents.ForceGetTaskResult(
                report_computed_task = report_computed_task,
            )
        )
```

Reply:
But why is the `response_type` `ForceGetTaskResult`? You should return exactly what the queue says, not another message that contains it. The spec does not even list `ForceGetTaskResult` as a valid value for response type.

Reply:
OK, repaired.